aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt

```diff
@@ -43,6 +43,7 @@ class AttributeTest {
         private val LIST_WITH_NON_EMPTY_ITEMS = "<ol><li a=\"B\">ab</li><li c=\"C\">c</li></ol>"
         private val LIST_WITH_EMPTY_ITEMS = "a<ul><li a=\"A\"></li><li></li><li a=\"1\">1</li><li></li><li b=\"B\"></li></ul>b"
         private val LIST_WITH_EMPTY_ITEMS_WITH_LINE_BREAK = "a<br><ul><li></li><li a=\"1\">1</li><li></li></ul><br>b"
+        private val LIST_WITH_EMPTY_ITEMS_WITH_LINE_BREAK_PARSED = "a<ul><li></li><li a=\"1\">1</li><li></li></ul><br>b"
```

Comment:
The [`markBlockElementLineBreaks`](https://github.com/wordpress-mobile/WordPress-Aztec-Android/blob/develop/aztec/src/main/kotlin/org/wordpress/aztec/AztecParser.kt#L150) method removes the preceding `<br>` by translating it into a `BlockElementLinebreak` span.